pages/index.tsx

```diff
@@ -72,6 +116,14 @@ export default function Home() {
         {error != '' && (
           <div className="w-full text-center text-sm text-red-500">{error}</div>
         )}
+        {hint != '' && (
+          <div
+            className="text-black-600 w-full cursor-pointer text-center text-base"
+            onClick={() => addDistrictFromHint(hint)}
+          >
+            Did you mean? <span className="font-semibold">{hint}</span>
+          </div>
+        )}
```

Comment:
empty string qualify as falsey in js, so the checks are redundant, please fix my checks too in the mean time